HomeTask2_3/Entities/Training.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace HomeTask2_3.Entities
+{
+    internal class Training
+    {
+        public string? Description { get; set; }
+
+        private TrainingElement[] trainingElements;
+        private int _itemCount;
+
+        public Training(string? description)
+        {
+            Description = description;
+            trainingElements = new TrainingElement[10];
+            _itemCount = 0;
+        }
+
+        public void Add(TrainingElement item)
+        {
+            if(_itemCount == trainingElements.Length)
+            {
+                Array.Resize(ref trainingElements, _itemCount * 2);
+            }
+
+            trainingElements[_itemCount] = item;
+            _itemCount++;
+        }
+
+        public bool IsPractical()
+        {
+            for (int i = 0; i < _itemCount; i++)
+            {
```

Comment:
better check  if (!(trainingElements[i] is Practice)), more reliable if other types of lessons will be added